web/angular.json

```diff
@@ -126,6 +126,10 @@
           }
         }
       }
-    }},
-  "defaultProject": "web"
+    }
+  },
+  "defaultProject": "web",
+  "cli": {
+    "analytics": "9d33d8b7-0859-4953-ba30-717f61e4aa31"
```

Comment:
Just curios, what is doing?

Reply:
Seems to be some automatic addition when the local screenshot testing takes place. I deleted it, then ran the local screenshot testing, and the analytics was added up again.

Reply:
Thanks for explanation.